packages/orchestration/src/chain-info.js

```diff
@@ -117,14 +115,8 @@ export const wellKnownChainInfo =
  * @param {(...messages: string[]) => void} log
  */
 export const registerChainNamespace = async (agoricNamesAdmin, log) => {
-  const { nameAdmin } = await E(agoricNamesAdmin).provideChild('chain');
-
-  const registrationPromises = Object.entries(wellKnownChainInfo).map(
-    async ([name, info]) => {
-      log(`registering chain ${name}`);
-      return E(nameAdmin).update(name, info);
-    },
-  );
-
-  await Promise.all(registrationPromises);
+  for await (const [name, info] of Object.entries(wellKnownChainInfo)) {
+    log(`registering chain ${name}`);
```

Comment:
in https://github.com/Agoric/agoric-sdk/pull/9477/ I found it helpful to be a little more specific since this log output doesn't have any context qualifier:
```suggestion
    log(`registering agoricName chain.${name}`);
```

Though now that there's a `registerChain` method the logging could go there.